src/Core/Settings.h

```diff
@@ -445,6 +445,8 @@ class IColumn;
     M(Bool, database_replicated_always_detach_permanently, false, "Execute DETACH TABLE as DETACH TABLE PERMANENTLY if database engine is Replicated", 0) \
     M(DistributedDDLOutputMode, distributed_ddl_output_mode, DistributedDDLOutputMode::THROW, "Format of distributed DDL query result", 0) \
     M(UInt64, distributed_ddl_entry_format_version, 1, "Version of DDL entry to write into ZooKeeper", 0) \
+    M(UInt64, external_storage_max_read_rows, DEFAULT_BLOCK_SIZE, "Limit maximum number of rows when MaterializeMySQL should flush history data. If equal to 0, this setting is disabled", 0) \
```

Comment:
```suggestion
    M(UInt64, external_storage_max_read_rows, DEFAULT_BLOCK_SIZE, "Limit maximum number of rows when table with external engine should flush history data. Now supported only for MySQL. If equal to 0, this setting is disabled", 0) \
```

Reply:
@TCeason, you forgot about this.

Reply:
`https://github.com/ClickHouse/ClickHouse/pull/22697/files#diff-aa3c979016a9f8c6ab5a51560411afa3f4cef55d34c899a2b1e7aff38aca4076R448`

M(UInt64, external_storage_max_read_rows, 0, "Limit maximum number of rows when MaterializeMySQL should flush history data. If equal to 0, this setting is disabled", 0) \

The default value of `external_storage_max_read_rows` has already modified.